### PR TITLE
Feature/amaretto component

### LINKED ISSUE #17: Iteration 2 - Specialty Drinks: Amaretto
GOAL: EOD Tuesday 

- [x] Basic SpecialsContainer to pass TS/props/etc.

- [x] API call for Amaretto drinks with filter/search functionality 

- [x] Render Amaretto drinks in AmarettoCards Component
---------------------------------------------------------
- [x] (NICE TO GET DONE) : ROUTER to get to next page

### PR DESCRIPTION
**Pull request by kendallm360**
## Summary:
Duplicate logic from martini component to make amaretto component. Add type file to dry up component files. Added additional routing for each component

## Type of Changes:
- [x] Bug fix
- [x] Refactor
- [x] New feature

## Where are the Changes:
All Files

## How was this tested:
Chrome dev tools
React dev tools

## What are the relevant tickets:
closes #17

### CODE REVIEW

File: src/components/MartiniDeck/MartiniDeck.tsx [hunk 1]
```diff
@@ -29,7 +29,7 @@ class MartiniDeck extends Component<{}, TypeState> {
     return this.state.drinkList.map((martini: Drink) => {
       //   console.log(martini);
       return (
-        <Link to={`/${martini.idDrink}`}>
+        <Link to={`/drinks/${martini.idDrink}`}>
```
Comment (jordan-sullivan) [inline src/components/MartiniDeck/MartiniDeck.tsx:32]: THE SOLUTION!!!!!!! Great thinking team to reach our for help when we were stuck. What a simple and effective solution. -Jordan

File: src/components/Types.ts [hunk 2]
```diff
@@ -9,6 +9,19 @@ export type Drink = {
   idDrink: string;
   strDrink: string;
   strDrinkThumb: string;
+  strInstructions?: string | null;
```
Comment (jordan-sullivan) [inline src/components/Types.ts:12]: We'll need to see if we can get the "extends" working either in the type or as an interface

**Review by jordan-sullivan — COMMENTED** [review]
Nice job getting through a tough iteration team! -J

File: src/components/Types.ts [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+// strIngredient3?: string | null;
+// strIngredient4?: string | null;
+// strIngredient5?: string | null;
+// strIngredient6?: string | null;
+// strIngredient7?: string | null;
+// strIngredient8?: string | null;
+// strIngredient9?: string | null;
+// strIngredient10?: string | null;
+// strIngredient11?: string | null;
+// }
```
Comment (nhodnett) [inline src/components/Types.ts:40]: We can delete these when we do the re-factoring.

**Review by nhodnett — APPROVED** [review]
All good crew, nice work on getting the data showing how we planned. Onto the weekend.

**Pull request merged**